### PR TITLE
feat: add transpileDependencies option

### LINKED ISSUE #104: How to add transpileDependencies in vue?
First of all i am really sorry for opening a new issue that is related to webpack config. This is related to vuetify.

## Description
A month pack i raised an issue about how to implement vuetify. You shared me a repo. That worked fine. Now i wanted to recreate it in a new project.

I created a new vue-cli project with vuetify. And i copy pasted all the files that vue-cli-plugin generated. One of the file it updated was vue.config.js to add `transpileDepencies` for vuetify. I dont know how to configure it in nx-plus/vue. Can you guide me please? 

Thank you. I am really sorry for this issue.

### PR DESCRIPTION
**Pull request by MatthieuLebigre**
## Current Behavior

`transpileDependencies` is not available

## Expected Behavior

Now that https://github.com/ZachJW34/nx-plus/pull/119 is merged, I made `transpileDependencies` available

## Related Issue(s)

Fixes #104

### CODE REVIEW

**Comment by ZachJW34** [comment]
Thanks for the PR! I'll start reviewing tonight.

File: libs/vue/src/builders/browser/schema.json [hunk 1]
```diff
@@ -64,6 +64,10 @@
       "description": "Setting this to false can speed up production builds if you don't need source maps for production.",
       "default": false
     },
+    "transpileDependencies": {
+      "type": "array",
+      "description": "By default babel-loader ignores all files inside node_modules. If you want to explicitly transpile a dependency with Babel, you can list it in this option."
```
Comment (ZachJW34) [inline libs/vue/src/builders/browser/schema.json:69]: Minor, but let's add `"default": []` so that it is inline with the vue cli.
https://cli.vuejs.org/config/#transpiledependencies

**Review by ZachJW34 — CHANGES_REQUESTED** [review]
Please add the same changes you made to the dev-server and browser builder to the library builder (libs/vue/src/builders/library) and update the [README](https://github.com/ZachJW34/nx-plus/blob/master/libs/vue/README.md) to show this new option for all three builders.
Also, how did you test this? I'm unfamiliar with this option so I'm not entirely sure what to test.

File: libs/vue/src/builders/browser/schema.json (hunk 1 above)
Comment (MatthieuLebigre) [reply libs/vue/src/builders/browser/schema.json]: done.

For testing it, I did in my project where it's needed. Here's the use case:
One of the dependencies uses a syntax not yet supported by default by Webpack (at least not the version that comes with vue-cli). In my case, that's https://github.com/ing-bank/lion/tree/master/packages/overlays that's uses some optional chaining

**Comment by MatthieuLebigre** [comment]
@ZachJW34 it should be good now. Do you mind create a release once it's merged ? I'd love to use this asap :)

File: libs/vue/README.md [hunk 2]
```diff
@@ -8,13 +8,33 @@
...
 ## Contents
 
-- [Prerequisite](#prerequisite)
-- [Getting Started](#getting-started)
-- [Nx Dependency Graph Support](#nx-dependency-graph-support)
-- [Schematics (i.e. code generation)](#schematics-ie-code-generation)
-- [Builders (i.e. task runners)](#builders-ie-task-runners)
-- [Modify the Webpack Configuration](#modify-the-webpack-configuration)
-- [Updating Nx Plus Vue](#updating-nx-plus-vue)
+- [Nx Plus Vue](#nx-plus-vue)
```
Comment (ZachJW34) [inline libs/vue/README.md:11]: Please undo these changes

File: libs/vue/README.md [hunk 3]
```diff
@@ -143,8 +163,6 @@ nx serve my-app
 
 ### Dev Server
 
-`nx serve <name> [options]`
```
Comment (ZachJW34) [inline libs/vue/README.md:146]: Please undo these changes.

**Review by ZachJW34 — CHANGES_REQUESTED** [review]

**Comment by ZachJW34** [comment]
Whenever you update, please squash your commits and then LGTM!!

File: libs/vue/README.md (hunk 2 above)
Comment (MatthieuLebigre) [reply libs/vue/README.md]: done

**Comment by MatthieuLebigre** [comment]
> Whenever you update, please squash your commits and then LGTM!!

All done @ZachJW34 !

**Review by ZachJW34 — APPROVED** [review]

**Comment by ZachJW34** [comment]
Thanks @MatthieuLebigre! We can publish another release soon so that you can use this option.
